assets/themes/campersites/campersites.json

```diff
@@ -0,0 +1,718 @@
+                ]
+              },
+              "then": {
+                "en": "This place has a sanitary dump station"
+              }
+            },
+            {
+              "if": {
+                "and": [
+                  "sanitary_dump_station=no"
+                ]
+              },
+              "then": {
+                "en": "This place does not have a sanitary dump station"
+              }
+            }
+          ]
+        },
+        {
+          "render": {
+            "en": "This place is operated by {operator}"
+          },
+          "question": {
+            "en": "Who operates this place?"
+          },
+          "freeform": {
+            "key": "operator"
+          }
+        },
+        {
+          "render": {
+            "en": "{capacity} campers can use this place at the same time"
+          },
+          "question": {
+            "en": "How many campers can stay here? (skip if there is no obvious number of spaces or allowed vehicles)"
+          },
+          "freeform": {
+            "key": "capacity",
+            "type": "pnat"
+          }
+        },
+        {
+          "render": {
+            "en": "internet access"
+          },
+          "question": {
+            "en": "Does this place provide internet access?"
+          },
+          "mappings": [
+            {
+              "if": {
+                "and": [
+                  "internet_access=yes"
+                ]
+              },
+              "then": {
+                "en": "There is internet access"
+              }
+            },
+            {
+              "if": {
+                "and": [
+                  "internet_access=no"
+                ]
+              },
+              "then": {
+                "en": "There is no internet access"
+              }
+            }
+          ],
+          "freeform": {
+            "key": "internet_access"
+          }
+        },
+        {
+          "render": {
+            "en": "internet access price"
+          },
+          "question": {
+            "en": "Do you have to pay for the internet access?"
+          },
+          "freeform": {
+            "key": ""
+          },
+          "mappings": [
+            {
+              "if": {
+                "and": [
+                  "internet_access:fee=yes"
+                ]
+              },
+              "then": {
+                "en": "You need to pay extra for internet access"
+              }
+            },
+            {
+              "if": {
+                "and": [
+                  "internet_access:fee=no"
+                ]
+              },
+              "then": {
+                "en": "You  do not need to pay extra for internet access"
+              }
+            }
+          ],
+          "condition": {
+            "and": [
+              "internet_access=yes"
+            ]
+          }
+        },
+        "reviews",
+        {
+          "render": {
+            "en": "toilets"
```

Comment:
Same issue as above: either allow a freeform text field and use the value, or remove them

Reply:
done